syntax = "proto3";

package mapper;

service Mapper {
  // Unary
  rpc SendFileLocation (MapperRequest) returns (MapperReply);

}
message MapperRequest {
  string file_name = 1;
  int64 number_of_reducers = 2;
}
message MapperReply {
}
